ldms/src/ldmsd/ldmsd_decomp.c

```diff
@@ -748,22 +748,22 @@ int ldmsd_row_to_json_object(ldmsd_row_t row, char **str, int *len)
 static const char *col_type_str(enum ldms_value_type type)
 {
 	static char *type_str[] = {
-	    [LDMS_V_CHAR] = "\"string\"",
-	    [LDMS_V_U8] = "\"string\"",
-	    [LDMS_V_S8] = "\"int\"",
+	    [LDMS_V_CHAR] = "{\"type\":\"int\",\"logicalType\":\"signed-char\"}",
```

Comment:
@morrone, should this be int8_t, or int8?

Reply:
I don't know. How about int8.

Reply:
done